docs/UsingKudos.rst

```diff
@@ -0,0 +1,63 @@
+.. _UsingKudos:
+
+
+############
+Using Kudos
+############
+
+
+General information about kudos
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can express your recognition to your colleagues by sending them kudos in order to thank them for their help, contributions and good work.
+You have a limited number of kudos that you are allowed to send per period of time. 
+
+If you don't use the kudos during the current period, you will not be able to use them in the next one. They are not cumulated with the number of kudos you are allowed to use in the new period.
+Both the number of kudos and periodicity are determined by the administrator of the platform.
+
+When you send a kudos you earn gamification points as well as the reciever. 
```

Comment:
```suggestion

```
This depends of a gamification rule that can be disabled. I would not mention it here.

Reply:
Ok. I deleted it.